src/common/prntbase.cpp

```diff
@@ -1976,6 +1982,18 @@ void wxPrintPreviewBase::InvalidatePreviewBitmap()
 
 bool wxPrintPreviewBase::UpdatePageRendering()
 {
+    static double s_sf = 0.0;
+
+    double newScaleFactor = m_previewCanvas->GetDPIScaleFactor();
+
+    if ( m_previewCanvas && newScaleFactor != s_sf )
```

Comment:
It doesn't make sense to check for a pointer we just dereferenced above. Either this check is not needed at all or it should be done first.

Reply:
Fixed.